backend/RoomPlanner/room-planner/src/main/resources/scripts/patch_2.sql

```diff
@@ -0,0 +1,42 @@
+drop table users;
```

Comment:
You should not drop the existent table.
You should alter it, meaning remove birth date column and add the new ones.